templates/compose/linkding-plus.yaml

```
@@ -0,0 +1,20 @@
+# documentation: https://linkding.link/
+# slogan: A self-hosted bookmark manager designed to be minimal, fast, and easy to set up. (Includes feature for archiving websites as HTML snapshots)
+# category: RSS
+# tags: rss, feed
+# logo: svgs/linkding.svg
+# port: 9090
+
+services:
+  linkding:
```

Change service name to make the compose name to avoid confusion
```suggestion
  linkding-plus:
```